src/mon/OSDMonitor.cc

```diff
@@ -8419,6 +8419,10 @@ int OSDMonitor::prepare_command_pool_set(const cmdmap_t& cmdmap,
     }
     p.min_size = n;
   } else if (var == "pg_num_actual") {
+    if (p.has_flag(pg_pool_t::FLAG_NOPGCHANGE)) {
```

Comment:
This commit seems to be unrelated to crimson. At first glance looks sane but maybe could be sent separately to facilitate broader review.